From qhfeng.kernel@gmail.com  Sun Jan 25 15:12:18 2009
From: Qinghuang Feng <qhfeng.kernel@gmail.com>
Date: Wed, 14 Jan 2009 15:45:13 +0800
Subject: SYSFS: use standard magic.h for sysfs
To: gregkh@suse.de
Cc: linux-kernel@vger.kernel.org
Message-ID: <496d980e.160d6e0a.14d0.ffffbe3d@mx.google.com>


SYSFS_MAGIC has been added into magic.h, so only use that definition
in magic.h to avoid potential consistency problem.

Signed-off-by: Qinghuang Feng <qhfeng.kernel@gmail.com>
Signed-off-by: Greg Kroah-Hartman <gregkh@suse.de>

---
 fs/sysfs/mount.c |    3 +--
 1 file changed, 1 insertion(+), 2 deletions(-)

--- a/fs/sysfs/mount.c
+++ b/fs/sysfs/mount.c
@@ -17,11 +17,10 @@
 #include <linux/pagemap.h>
 #include <linux/init.h>
 #include <linux/module.h>
+#include <linux/magic.h>
 
 #include "sysfs.h"
 
-/* Random magic number */
-#define SYSFS_MAGIC 0x62656572
 
 static struct vfsmount *sysfs_mount;
 struct super_block * sysfs_sb = NULL;
